### PR TITLE
Gh 512 save remove pk

### PR DESCRIPTION
**Pull request by sche**

### CODE REVIEW

File: Multisig/Cross-layer/App.swift [hunk 1]
```diff
@@ -9,43 +9,44 @@
...
 
 class App {
-
     static let shared = App()
 
     // UI layer
-    var viewState = ViewState()
-    var theme = Theme()
-    var snackbar = SnackbarCenter()
+    let viewState = ViewState()
```
Comment (MouazAlzahabi) [inline Multisig/Cross-layer/App.swift:15]: nice

File: Multisig/Cross-layer/App.swift [hunk 2]
```diff
@@ -9,43 +9,44 @@
...
+    let ens = ENS(registryAddress: "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e")
+    let gnosisSafe = GnosisSafe()
 
     // Data Layer
 
-    // Services    
-    var safeTransactionService = SafeTransactionService(
+    let keychainService = KeychainService(identifier: App.configuration.app.bundleIdentifier)
+
+    // Services    ∫
```
Comment (MouazAlzahabi) [inline Multisig/Cross-layer/App.swift:30]: ∫ what is this for?

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift [hunk 3]
```diff
@@ -14,22 +14,41 @@ struct BasicAppSettingsView: View {
...
+    }
+
     @State
     var addOwnerIsActive = false
 
     var body: some View {
         List {
-            ForEach(0..<1) { _ in
+            if signingKeyAddress != nil {
+                ForEach(0..<1) { _ in
```
Comment (MouazAlzahabi) [inline Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift]: Why do we have ForEach here, you can directly define the content

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift [hunk 4]
```diff
@@ -14,22 +14,41 @@ struct BasicAppSettingsView: View {
...
+                    VStack(alignment: .leading, spacing: 4) {
+                        Text("Signing key").bold()
```
Comment (MouazAlzahabi) [inline Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift]: Maybe we can use .title() style that we already define?

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift [hunk 5]
```diff
@@ -14,22 +14,41 @@ struct BasicAppSettingsView: View {
...
-                               isActive: self.$addOwnerIsActive) {
-                    Text("Import owner wallet").body()
-                }
-                .isDetailLink(false)
-            }
-            .onDelete { _ in
-                // Do nothing for now
-            }
-
+//            if signingKeyAddress != nil {
```
Comment (MouazAlzahabi) [inline Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift:29]: Commented code?

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/Cross-layer/App.swift (hunk 2 above)
Comment (sche) [reply Multisig/Cross-layer/App.swift:30]: some typo :)

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift (hunk 3 above)
Comment (sche) [reply Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift]: this is needed to enable swipe to delete

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift (hunk 4 above)
Comment (sche) [reply Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift]: no design yet, this is a placeholder

File: Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift (hunk 5 above)
Comment (sche) [reply Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift:29]: disabled for this release

**Pull request merged**